### ISSUE TITLE
tensorboard embeddings

### ISSUE THREAD

**Issue by wgmueller1** [bkj/tdesc#3]
Are you interested in adding a script to produce tensorboard embeddings from the output of tdesc?

I've written a script which creates the image sprites and formats the output of tdesc.
https://www.tensorflow.org/versions/r0.12/how_tos/embedding_viz/

**Comment by bkj** [comment]
Yeah that would be pretty cool.  I'd just set it up something like:

```
find ./data -type f | python -m tdesc > feats
python tdesc2tensorboard.py --feats feats
```

**Comment by wgmueller1** [comment]
Cool, let me generalize it a little so you can add labels as well.
